pkgs/applications/misc/wlclock/default.nix

```diff
@@ -0,0 +1,46 @@
+{ pkgs
+, stdenv
+, lib
```

Comment:
```suggestion
{ lib
, stdenv
```
`pkgs` isn't used and lib should always be first

Reply:
Thanks! I totally missed that xD